packages/entities/entities-gateway-services/src/components/GatewayServiceForm.vue

```diff
@@ -569,9 +670,7 @@ const handleFloatVal = (oVal: string) => {
   return 0
 }
 
-const changeCheckedGroup = () => {
-  isCollapsed.value = true
-  form.errorMessage = ''
+const initFieldDefaultValues = () => {
```

Comment:
```suggestion
const initFieldDefaultValues = (): void => {
```

nit: void return type